Add headings and 2i approved text to release doc (#2443)
alphagov/govuk-frontend: docs/releasing/publishing.md

11. Once a reviewer approves the pull request, merge it to **main**.

## Publish a release to npm

1. Check out the **main** branch and pull the latest changes.

> 2. Log into npm (`npm login`), using the npm/govuk-patterns-and-tools team [credentials](https://github.com/alphagov/design-system-team-credentials/tree/main/npm/govuk-patterns-and-tools).

Reviewer: Can we update https://github.com/alphagov/govuk-frontend/blob/main/docs/releasing/publishing-from-a-support-branch.md to match?

Author: Definitely! I'm also going to change 'log into' to 'sign in to', based on [GOV.UK style guide advice](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#sign-in-or-log-in):

> Use sign in rather than log in (verb) for calls-to-action where users enter their details to access a service.

```suggestion
2. Sign in to npm (`npm login`), using the npm/govuk-patterns-and-tools team [credentials](https://github.com/alphagov/design-system-team-credentials/tree/main/npm/govuk-patterns-and-tools).
```
